mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -1449,7 +1449,7 @@ func TestAccContainerCluster_withMaintenanceExclusionWindow(t *testing.T) {
 				ImportStateVerify:   true,
 			},
 			{
-				Config: testAccContainerCluster_withExclusion_DailyMaintenanceWindow(cluster, "2020-01-01T00:00:00Z", "2020-01-02T00:00:00Z", "2020-05-01T00:00:00Z", "2020-05-02T00:00:00Z"),
```

Comment:
Can you add an update case? Ideally we'd go from 2 -> 1.

Reply:
Explained above